### DISCUSSION TITLE
Nightly Builds

### DISCUSSION THREAD

**Discussion by moosehadley** [djsime1/awesome-flipperzero#15] [Ideas]
Hello!

I've set up a build bot that makes nightlies of several of the popular firmwares.

I'm not sure if you want to add it, or where it would go on the list.
But I thought I'd offer!

https://github.com/moosehadley/Flipperzero-Nightly-Builds

**Comment by djsime1** [comment]
The stock firmware repo already has nightly builds, and the page 503'd me a few times when I tried to visit. I have my own private build server already set up and I was thinking about exposing builds along with logs that automatically trigger on commit.
